packages/ui/lib/RichEdit/index.js

```diff
@@ -41,11 +41,15 @@ function RichEdit(
     setContent,
     onSubmit = () => {},
     showButtons = true,
-    showOkButton = true,
+    showSubmitButton = true,
```

Comment:
its too many props here, beside ,we are still not able to override `className` & `onClick`

I would suggest that we merge these props into one, `submitButton`/`previewButton`

pesudo below
```
const submitButton = <>...</>
const previewButton = <>...</>


function RichEdit(
  {
    content,
    setContent,
    onSubmit = () => {},
    submitButton = submitButton,
    previewButton = previewButton,
```

if somebody wants to hide one of the buttons, just do 
`<RichEditor submitButton={null}>`

also, this allows override button element completely by passing `submitButton={<>anything here</>}`

Reply:
looks good, but I am afraid we can't, read this code in line 93 `setPreview(false)`:

```js
            <Button
              className="button-submit"
              {...submitButtonProps}
              primary
              isLoading={submitting}
              disabled={disabled}
              onClick={() => {
                setPreview(false);
                onSubmit();
              }}
            >
```

the `preview` state is manage by component itself, not from outside.

Reply:
it will be fine, just warp `submitButton` with a `<SubmitButtonWrapper onClick={() => {                setPreview(false);                 onSubmit();              }}>`

Im fine with delaying this change and accept the PR first